packages/vuetify/dev/vuetify.js

```diff
@@ -5,18 +5,30 @@ import { aliases, mdi } from 'vuetify/src/iconsets/mdi'
 import { fa } from 'vuetify/src/iconsets/fa-svg'
 import { ar, en, ja, sv } from 'vuetify/src/locale'
 import * as directives from 'vuetify/src/directives'
+import { VuetifyDateAdapter } from 'vuetify/src/labs/date/adapters/vuetify'
+import datefnssv from 'date-fns/locale/sv'
+import datefnsen from 'date-fns/locale/en-US'
+import DateIoDateFnsAdapter from '@date-io/date-fns'
 
 export default createVuetify({
   directives,
   ssr: !!process.env.VITE_SSR,
   locale: {
+    locale: 'en-US',
     messages: {
-      en,
+      'en-US': en,
```

Comment:
Should this be part of the locale object instead somehow?

Reply:
Maybe match `navigator.languages` with the key as a prefix? Mine is `['en-AU', 'en-GB', 'en-US', 'en']` so with `locale: 'en'` my formatting would be `en-AU`

Reply:
This guy says that's a bad idea though https://blog.sffc.xyz/post/190943794505/why-you-should-not-parse-localized-strings#:~:text=User%20Language%20!%3D%3D%20Device%20Language

Reply:
As a non-English native, I totally agree. I have all my devices set to English for the language but I'm using a 24 hour clock and the formatting of time, dates, currency and so on are all set to my native language.

Reply:
What's `navigator.languages` for you?

Reply:
For Safari (which I usually use) it's `["en-US"]`. For Chrome it's `['en-US', 'en', 'sv']`.

Reply:
And `new Date(1999, 1, 1).toLocaleDateString()`?

Reply:
`"2/1/1999"`.

Reply:
Here my `navigator.languages` are `de-DE,de,en-US,en`
`new Date(1999, 1, 1).toLocaleDateString()` gives me `1.2.1999`

Reply:
Yeah I have en-AU with en-SE for dates, but `<input type="date">` and `toLocaleDateString` just use en-AU.
I think we have three options here:
1. display and accept only ISO8601 like in v2
2. parse the result of `new Date(1999, 1, 1).toLocaleDateString()` internally and use it to infer a format. this could get pretty complicated though
3. either [1] or [2] but with `formatIn`/`formatOut` props to override it with something like moment.js that does support localisation

Reply:
For `<input type="time">` Chrome seems to follow my date time settings, but not Safari. Safari follow the language setting.